spec/uo_cli_spec.rb

```diff
@@ -156,12 +156,21 @@ def delete_directory_or_file(dir_or_file)
     end
 
     it 'creates scenario visualization for default post processor' do
-      filename = File.join(test_directory, 'run', 'two_building_scenario', 'default_scenario_report.csv')
+      @root_dir = test_directory
       system("#{call_cli} process --default --scenario #{test_scenario} --feature #{test_feature}")
-      system("#{call_cli} process --visualize --scenario #{test_scenario} --feature #{test_feature}")
+      system("#{call_cli} process --visualize-scenarios --scenario #{test_scenario} --feature #{test_feature}")
       expect(File.exist?(File.join(test_directory, 'run', 'scenario_comparison.html'))).to be true
     end
 
+    it 'creates feature visualization for default post processor' do
+      @root_dir = test_directory
+      @scenario_file_name = "two_building_scenario"
```

Comment:
Are lines 159, 166 & 167 necessary?

Reply:
https://github.com/urbanopt/urbanopt-cli/pull/122/files#diff-e4f12bf02f874628bd78d9cc86f25c58R475 and https://github.com/urbanopt/urbanopt-cli/pull/122/files#diff-e4f12bf02f874628bd78d9cc86f25c58R483 are using it for creating the visualization